config/locales/en.yml

```diff
@@ -841,6 +843,7 @@ en:
         agfs_hardship_hint: For advance payments in exceptional circumstances
         agfs_interim_rb: Advocate warrant fee
         agfs_interim_hint: For warrant fee claims with the earliest representation order dated on or after 1 April 2018
+        lgfs_hardship_hint_html: For advance payments where PTPH is yet to take place, or has been adjourned for a further PTPH/case management hearing see <a href="#" target="_blank" rel="external" title="COVID 19 Remuneration Regulations">COVID 19 Remuneration Regulations</a> for full details
```

Comment:
can do afterwards but i have just put the expected publishing site in here
https://www.gov.uk/guidance/financial-relief-for-legal-aid-practitioners